nodepool/files/nodepool.yml

```diff
@@ -53,6 +53,8 @@ providers:
         config-drive: true
     pools:
       - name: main
+        # Ignore the quota given by public cloud and only respect what we provide as max-servers
+        ignore-provider-quota: True
```

Comment:
Not sure if this will matter - I doubt it - but all other booleans are lower case.

Reply:
👍 fixed